plugins/module_utils/modules.py

```diff
@@ -0,0 +1,20 @@
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.common import set_collection_info
+
+
+COMMUNITY_AWS_COLLECTION_NAME = "community.aws"
+COMMUNITY_AWS_COLLECTION_VERSION = "6.0.0-dev0"
```

Comment:
For consistency should we bump this over into `plugins/module_utils/common.py`?

Reply:
ok will do, Thanks